.vscodeignore

```diff
@@ -1,4 +1,11 @@
 .vscode/**
 .vscode-test/**
 .gitignore
-vsc-extension-quickstart.md
+python/**
+node_modules/**
```

Comment:
Is it okay to ignore even the (runtime) dependencies? Does webpack include them in the `dist` version?

Reply:
Never mind, I read more :)